### DISCUSSION TITLE
Regarding ChromieCraft News 28-Nov-2022

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4512] [Suggestions]
Hi,

ref: https://www.chromiecraft.com/en/the-burning-crusade-on-chromiecraft/
progression: https://www.chromiecraft.com/en/progression/

first thank you for bringing some more clarification into the release schedule.
Many points are noted and understandable for the sake of the dev team to not having burnout-like experience.

Still, i would like to address a few more points missing or to be revised potentially.

#### Onyxia
what about her and her tuning / disappearance? the last statement I am aware of is that Ony will stay as is and only be removed at the end of tbc.

#### TBC World bosses
Kazzak and Doomwalker will be the two outdoor world bosses relevant up to t5 content for most.
Their loot used to be **bind on pickup** until Sunwell. Afterwards, and thats the current default here, too, it is **bind on equip**. Can we assume this gonna be changed, too?
https://wowwiki-archive.fandom.com/wiki/Talon_of_the_Tempest

#### Attunments
Quoting from the announcement:
> Attunements will be required as they were in TBC, however only for the current raid. Once we progress to the next raid, the previous raid will no longer require you to be attuned to it in order to run the raid.

Looking at the progression, this would mean that we do not need SSC attunement once TK releases and we do not need Hyjal attunement as soon as Black Temple releases.
Is this meant this way or is it supposed to be for the **tier content**, not the **next raid**?

#### Darkmoon Cards
The cards were added only at [Patch 2.1](https://wowpedia.fandom.com/wiki/Darkmoon_Card:_Vengeance), which is the Black Temple Patch. Given their item level (100), a release with t5 content seems a bit more reasonable alongside the factions for Ogrila and Shatari.
Currently, there is nothing stated about removal or implementation yet.

#### Blue Level 70 PvP Set
this set used to be accessible via honor, and late into TBC it was added for honored reputation at the different faction npcs. I think it is safe to assume that this wont be available for honored reputation and will be stated explicitly with the upcoming announcement regarding PvP.

#### Black Morass
Two of the three (and hence the majority) of the bosses provide loot for level 68. Further, this dungeon is one of the places used by many to AoE grind, level and farm leather ([see youtube](https://www.youtube.com/results?search_query=black+morass+grind)). Not having it accessible at 66, would be a shame from the point of view of offering a retail-like experience.

#### Gated, Parallel Raid Content Release
Blizzard did this already a few times and it might be an option to consider instead, too:
Releasing raid content gradually, meaning e.g. for T5 content: We get access to both raids in week 1, with the first boss each. And then gradually add encounters (e.g. 1 boss every week, sometimes more than 1, sometimes none).
The advantage of this concept would be that the devs have a reduced amount of bosses to focus on instead of getting e.g. a pile of 9 bosses to fix in parallel upon release of Black Temple. And the users would get new content on a gradual basis. The disadvantage obviously would be that guilds couldn't really compete for a server first any more, except if the "heroic" mode is delayed until all normal mode bosses are released (which would likely not be favored).
T4 and ZA might be special in this context because of the raid size, but maybe it is something to try out at t5 content and see if it proves fine for further implementations of raid content.

#### Level Brackets
with the "high" level dungeons dropping mostly level 68 and 70 gear, an option would be to limit leveling to 68 instead of 69 and having people level 2 levels instead of one for the final release as it was the case with 59 to 60.
No real advantage of disadvantage in my opinion, but maybe I am also having an oversight on a few aspects.

#### Mage portals
I understand that most will be able to put their hearthstones to Shattrath. But, especially from the point of view as a potential new joiner to the server, the portals in and between the capital cities proved a nice addition and valuable to low level players.
I personally would prefer seeing them stay as is - without adding or removing any other portals.

**Comment by Cooties-cc** [comment]
Seems some thoughts I had are here, so guess I'll just wait for comment on them. 

However for mage portals, I think a better solution was to remove the other city portals, and just add a shat portal to org/SW. Downsides would be a) direct path to shat instead of having people run there like originally, b) low level players could enter a zone much higher than normal (however warlocks can summon lowbies to outland, and mages can port them). The advantages are a) players now can move more freely between TBC and Vanilla content, b) lowbies can use the shat portals to move freely between cities like before with fewer port bloat in main cities.

**Reply by AcetylGer** [reply]
You could just lvl-restrict the shatt portal to 64 the same way the blasted lands portal is lvl- restricted.
That way new players still have to run to shatt at lvl 60 and players at the cap of the first bracket can move between shatt and old world easily.

**Reply by balleny** [reply]
that sounds very reasonible

**Comment by balleny** [comment]
Also, by locking Botanica and Black Morass, 2 out of 3 Alchemist specializations are only accessible indirectly by getting 3 Primal Mights and then changing the specialization by paying 150g (besides the fact that Primal Mights gonna be a lot more expensive in the first place to get a specialization)

**Reply by Cooties-cc** [reply]
I semi forgot about this too, when guildmates were talking about having alt alchs. I had told them to do the easier route for choosing a special, but completely spaced that the other 2 were locked behind the quest items. I also wanted to add that it was 4 mights to complete the quest, https://wowgaming.altervista.org/aowow/?quest=10899, which will only take 4 days, but a large loss of materials early on. 

To be clear too, the quests require level 68, so this wasn't an issue for the next phase, but the one after. Allowing Black Morass doesn't seem like a huge issue for the 69 bracket, thou some parts of the kara attune I think are allowed here, and half the gear is for level 70 (cant use it anywise till next bracket). Botanica is also not that bad, but the half problem with that is that you need flying to normally access the instance, and the other half is all the gear is for level 70 lol. However you can most likely just que RDF to enter without flying.
